publication-commons/src/main/java/no/unit/nva/publication/permissions/file/FileStrategyBase.java

```diff
@@ -106,4 +108,20 @@ protected boolean isExternalClientWithRelation() {
                userInstance.isExternalClient() &&
                userInstance.getCustomerId().equals(resource.getPublisher().getId());
     }
+
+    protected boolean isDegree() {
+        return resource.isDegree();
+    }
+
+    public boolean isEmbargo() {
+        return !file.getFile().fileDoesNotHaveActiveEmbargo();
+    }
```

Comment:
Do not like that we negate two times in here, maybe File have method hasActiveEmbargo? But is not the method you added, so probably fine.

Reply:
yup will do, 3 of 5 uses had ! so we should turn it around